etp-backend/src/main/clj/solita/etp/service/energiatodistus_search.clj

```diff
@@ -168,7 +168,9 @@
   (cond
     (or (rooli-service/paakayttaja? whoami)
         (rooli-service/laskuttaja? whoami))
-    ["energiatodistus.tila_id IN (2, 3, 4)"]
+    ["(energiatodistus.tila_id IN (2, 3, 4) OR
+       (energiatodistus.draft_visible_to_paakayttaja = true AND
```

Comment:
boolean arvoa ei tarvitse vertailla true-literaaliin

Reply:
eli ihan samalla tavalla kuin clojuressakin: draft-visible-for-paakayttaja? === (= draft-visible-for-paakayttaja? true)

Reply:
Muutettu.